packages/devtools_app/lib/src/memory/memory_vm_chart.dart

```diff
@@ -164,11 +164,11 @@ class MemoryVMChartState extends State<MemoryVMChart> with AutoDisposeMixin {
 
   // TODO(terry): Move colors to theme?
   static final capacityColor = Colors.grey[400];
-  static final externalColor = Colors.blue[400];
-  static const rasterLayerColor = Color(0xff99cc00); // HoloGreenLight
-  static const rasterPictureColor = Color(0xffff4444); // HoloRedLight
-  static const rssColor = Color(0xffffbb33); // HoloOrangeLight
-  static const usedColor = Color(0xff33b5e5); // HoloBlue
+  static const usedColor = Color(0xff33b5e5);
```

Comment:
Would be good to get some uxr feedback on these new colors.

Reply:
I presume you talking about the raster cache colors only?
I'll create an issue for this https://github.com/flutter/devtools/issues/2616